modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -1,4 +1,5 @@
 = User Authentication with LDAP
+:experimental:
```

Comment:
this is a "bonus" line that is not in the master PR ???

Reply:
I must have been testing and left that in by accident. Thanks for spotting it.

Reply:
Removed.